package.json

```diff
@@ -12,6 +12,8 @@
   },
   "version": "0.1.0",
   "devDependencies": {
+    "@prettier/plugin-ruby": "^2.0.0",
```

Comment:
question: Why not use rubocop?

Reply:
I am more familiar with prettier 😅 Not very familiar with Ruby tooling yet. Does Rubocop have more flexibility over Prettier?

Reply:
No. Its just a tool focused for ruby & has support for rails. Please use that instead of prettier